homeworks/02-image-layout-draft1/main.css

```diff
@@ -0,0 +1,10 @@
+.imageContainer {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  gap: 3px;
+}
+
+.imageContainer img {
+  height: 220px;
```

Comment:
this would be too easy :wink:

The goal is to preserve the aspect ratio for an image :)

Reply:
That's why it's draft. Hard way is in 02-image-layout folder.